shortbread-website/content/customizing/_index.md

```diff
@@ -0,0 +1,141 @@
+
+Vector tiles are usually rendered using [MapLibre styles](https://maplibre.org/maplibre-style-spec/).
+You can either write your style in this JSON format or use a visual style editor like [Maputnik](https://maplibre.org/maputnik/).
+
+## Extending Shortbread
+
+### Creating custom Shotbread tiles
+
+See [Creating Shortbread Vector Tiles](/make-vectortiles/) for workflows and tutorials to create custom Shortbread tiles.
+
+The main options for custom configurations are:
+* tilemaker config (JSON, Lua)
+* Planetiler config (YAML, Java)
+* osm2pgsl config (Lua)
+
+### Basic rules
+
+* Keep tile size small
+* Don't break base map styles
+* Discuss core extensions in [shortbread-docs repo](https://github.com/shortbread-tiles/shortbread-docs)
+
+### Tileset combinations
+
+There are pros and cons for creating custom base map tiles or creating additional tilesets:
+
+* Extended base map tiles
+  * No additional tiles
+  * Can break base map styles
+  * Selfhosting required
+* Additional tilesets
+  * Additional tile source in style
+  * No easy relation to base map tiles
+  * Potential duplication of data
+
+### Server-side combination
+
+Mapbox vector tiles allow concatenating tiles on server-side.
+
+There are some drawbacks though:
+* Requires a service
+* Not supported by all tile servers
+* Caching more difficult
+
+## Publish custom tiles
+
+There are many ways to publish you own maps based on Shortbread tiles. Here's a minimal tutorial using [PMTiles](https://docs.protomaps.com/pmtiles/) for storing tiles. Tiles are efficently stored in a single file and no additional server software is required for serving tiles. The only requirement is an HTTP server supporting HTTP range requests.
+
+### Preparing tiles and style
+
+```bash
+# Generate Shortbread tiles
+docker run --rm --user=$UID -v $PWD/data:/data ghcr.io/onthegomap/planetiler shortbread.yml \
+  --download --area=liechtenstein --output=/data/shortbread.pmtiles
+# Download Versatiles Styles + Assets
+mkdir -p styles assets/sprites
+wget -O - https://github.com/versatiles-org/versatiles-style/releases/latest/download/styles.tar.gz | tar xz -C styles
+wget -O - https://github.com/versatiles-org/versatiles-style/releases/latest/download/sprites.tar.gz | tar xz -C assets/sprites
+# Replace tile source with PMTiles URL
+jq '.sources."versatiles-shortbread".url="pmtiles:///shortbread.pmtiles" | del(.sources."versatiles-shortbread".tiles)' \
+   styles/colorful.json > styles/colorful-pmtiles.json
+# Make sprite URL relative
+sed --in-place -e 's!https://tiles.versatiles.org/assets/sprites!/assets/sprites!g' styles/*.json
+```
+
+### Viewer HTML
+
+index.html:
+```html
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>MapLibre Viewer</title>
+    <script src='https://unpkg.com/maplibre-gl@4.5.0/dist/maplibre-gl.js'></script>
+    <link href='https://unpkg.com/maplibre-gl@4.5.0/dist/maplibre-gl.css' rel='stylesheet' />
+    <script src="https://unpkg.com/pmtiles@3.0.6/dist/pmtiles.js"></script>
```

Comment:
It's better to not pin these to an exact version number. Anything with the same major version should work.